spec/app_config.yml

```diff
@@ -19,6 +19,8 @@ default: &defaults
   tax_default: 6.0
   email_sender: noreply@minimal.test
 
+  host: localhost
+
```

Comment:
Why is this necessary?

Reply:
newer rails versions will throw an error if the host is not set